deployment/terraform/utils.go

```diff
@@ -173,3 +173,30 @@ func (t *Terraform) getStatePath() string {
 	}
 	return statePath
 }
+
+func (t *Terraform) getParams() []string {
```

Comment:
Could we use this in `*Terraform.Destroy()`?

Reply:
Oops, totally missed that one. I swear I checked the code :stuck_out_tongue: